components/pkg-export-container/src/cli.rs

```diff
@@ -31,10 +38,10 @@ pub fn cli() -> Command {
                                                      .value_name("IMAGE_NAME")
                                                      .help("Image name template: supports: \
                                                             {{pkg_origin}}/{{pkg_name}} \
-                                                            {{pkg_origin}}, {{pkg_name}}, \
-                                                            {{pkg_version}}, {{pkg_release}}, \
-                                                            {{channel}})")
-                                                     .default_value("{{pkg_origin/pkg_name}}"));
+                                                            (default), {{pkg_origin}}, \
+                                                            {{pkg_name}}, {{pkg_version}}, \
+                                                            {{pkg_release}}, {{channel}})")
+                                                     .value_parser(SUPPORTED_IMAGE_NAME_TEMPLATES));
```

Comment:
This validation looks incorrect. As I understand it, this enforces that the value of `--image-name` must be one of the values of `SUPPORTED_IMAGE_NAME_TEMPLATES`, which is not what we want. The template could contain any text and promises to treat each of the tokens in `SUPPORTED_IMAGE_NAME_TEMPLATES` as substitutable for their effective values. For example `{{pkg_name}}-MY_ORG` should be a valid template while this does not allow it.

Reply:
Oh okay, I was referring to `help` of that option and thought only those are supported. Removed that validation now.